src/views/hooks/useProjectAccessKey.ts

```diff
@@ -0,0 +1,7 @@
+export const useProjectAccessKey = () => {
+  const projectAccessKey =
+    import.meta.env.VITE_PROJECT_ACCESS_KEY ||
+    "AQAAAAAAADVH8R2AGuQhwQ1y8NaEf1T7PJM";
```

Comment:
We no longer need these || defaults inlined like this, because of the way we copy .env.example to .env when we clone the repo

Reply:
Done